v0/destinations/intercom/transform.js

```diff
@@ -95,23 +95,29 @@ function validateAndBuildResponse(message, payload, category, destination) {
       response.body.JSON = removeUndefinedAndNullValues(
         validateIdentify(message, payload)
       );
+      response.headers = {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${destination.Config.apiKey}`,
+        Accept: "application/json",
+        "Intercom-version": "1.4"
+      };
       break;
     case EventType.TRACK:
       response.body.JSON = removeUndefinedAndNullValues(
         validateTrack(message, payload)
       );
+      response.headers = {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${destination.Config.apiKey}`,
+        Accept: "application/json"
+      };
       break;
     default:
       throw new CustomError("Message type not supported", 400);
   }
 
   response.method = defaultPostRequestConfig.requestMethod;
   response.endpoint = category.endpoint;
-  response.headers = {
-    "Content-Type": "application/json",
-    Authorization: `Bearer ${destination.Config.apiKey}`,
-    Accept: "application/json"
-  };
```

Comment:
We should add the version header for all the requests, not just for users endpoint.

Reply:
Only /users endpoint was failing since its no longer available. Should we add version for track calls as well?

Reply:
Yes